### PR TITLE
Adds unit tests for platform api functions

### PR DESCRIPTION
**Pull request by jrabbott**
Adds unit tests for platform api functions

### CODE REVIEW

File: platform/tests/EducationBenchmarking.Platform.Tests/Benchmark/WhenFunctionReceivesPostComparatorSetRequest.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        var result =
+            await Functions.PostComparatorSet(CreateRequestWithBody(new ComparatorSetRequest())) as JsonContentResult;
+
+        Assert.NotNull(result);
+        Assert.Equal(200, result?.StatusCode);
+    }
+
+    [Fact]
+    public async Task ShouldReturn40OOnInvalidRequest()
```
Comment (Faizi52996) [inline platform/tests/EducationBenchmarking.Platform.Tests/Benchmark/WhenFunctionReceivesPostComparatorSetRequest.cs:33]: Typo in name. should be ShouldReturn400OnInvalidRequest()

Comment (jrabbott) [reply platform/tests/EducationBenchmarking.Platform.Tests/Benchmark/WhenFunctionReceivesPostComparatorSetRequest.cs]: Nice spot - push a fix

File: platform/tests/EducationBenchmarking.Platform.Tests/Establishment/WhenFunctionReceivesSuggestOrganisationsRequest.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        var result =
+            await Functions.SuggestOrganisationsAsync(CreateRequestWithBody(new PostSuggestRequest())) as JsonContentResult;
+
+        Assert.NotNull(result);
+        Assert.Equal(200, result?.StatusCode);
+    }
+
+    [Fact]
+    public async Task ShouldReturn40OOnInvalidRequest()
```
Comment (Faizi52996) [inline platform/tests/EducationBenchmarking.Platform.Tests/Establishment/WhenFunctionReceivesSuggestOrganisationsRequest.cs:32]: there is a typo in the method name in 400.

Comment (jrabbott) [reply platform/tests/EducationBenchmarking.Platform.Tests/Establishment/WhenFunctionReceivesSuggestOrganisationsRequest.cs]: Pushed a fix

File: platform/tests/EducationBenchmarking.Platform.Tests/Establishment/WhenFunctionReceivesSuggestTrustsRequest.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+        var result =
+            await Functions.SuggestTrustsAsync(CreateRequestWithBody(new PostSuggestRequest())) as JsonContentResult;
+
+        Assert.NotNull(result);
+        Assert.Equal(200, result?.StatusCode);
+    }
+
+    [Fact]
+    public async Task ShouldReturn40OOnInvalidRequest()
```
[the remark this answers is not in the document]
Comment (jrabbott) [reply platform/tests/EducationBenchmarking.Platform.Tests/Establishment/WhenFunctionReceivesSuggestTrustsRequest.cs]: Pushed a fix

**Review by Faizi52996 — APPROVED** [review]

**Review by J05h-L — APPROVED** [review]

**Pull request merged**